frontend/lib/src/components/elements/Expander/Expander.tsx

```diff
@@ -45,10 +117,13 @@ const Expander: React.FC<ExpanderProps> = ({
   children,
 }): ReactElement => {
   const { label, expanded: initialExpanded } = element
-
-  const [expanded, setExpanded] = useState<boolean>(initialExpanded)
+  const [expanded, setExpanded] = useState<boolean>(initialExpanded || false)
   useEffect(() => {
-    setExpanded(initialExpanded)
+    // Only apply the expanded state if it was actually set in the proto.
+    if (notNullOrUndefined(initialExpanded)) {
+      console.log("Setting expanded state to", initialExpanded)
```

Comment:
console statement should be removed

Reply:
upsi :( would love to have some linter for those console and print statements